src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -259,7 +259,7 @@ private void ConfigureLanguageServer(IServiceCollection serviceCollection)
         serviceCollection.AddLogging(logging =>
         {
             logging.AddFilter<LogHubLoggerProvider>(level => true);
-            logging.AddProvider(_loggerProvider);
+            logging.AddProvider(_loggerProvider!);
```

Comment:
The `AssumeNotNull()` helper will fix the nullability and throw with a decent message if `_loggerProvider` is null.

```suggestion
            logging.AddProvider(_loggerProvider.AssumeNotNull());
```